openspec/changes/archive/2026-04-09-governance-02-github-hierarchy-cache/specs/agile-feature-hierarchy/spec.md

```
@@ -0,0 +1,22 @@
+## MODIFIED Requirements
+
+### Requirement: GitHub Agile Feature Hierarchy
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)